### ISSUE TITLE
Undefined Array key 'title'

### ISSUE THREAD

**Issue by dchymko** [statamic/importer#4]
### Bug description

I'm getting this error when trying to import an export from WordPress using the default WordPress exporter (XML)

### How to reproduce

1. Create new empty collection and open Importer.
2. Select XML File (WP built-in export: https://drive.proton.me/urls/5CPMCZVA0G#ZeuCfcu8mVho, WP-All export: https://drive.proton.me/urls/KM68HW3XH8#dCj7WNxXBmHL)
3. Set all 3 fields to use the "Title" field from the import file
4. Choose 'slug' for the Unique Field radio button
5. Click 'save and run'
6. Error appears on screen as `Undefined array key 'title'`
7.  Change unique field to be 'title'
8. Click 'Save and run', see same error

### Environment

```yaml
Statamic 5.35.0
Importer 1.0.1
PHP 8.3.6

Running locally using `php artisan serve`
```

**Comment by dchymko** [comment]
Hmmm, this also seems to happen when I use an XML file generated from WP ALL Import (added link to step #3). Maybe its an issue with blank/empty post titles

**Comment by duncanmcclean** [comment]
I can't seem to reproduce this issue after fixing #5.

Are you able to let me know if it's fixed after updating to [v1.0.2](https://github.com/statamic/importer/releases/tag/v1.0.2)?

**Comment by dchymko** [comment]
Yup, both issues seem to be fixed, thanks!
